### ISSUE TITLE
Faulty UTXO selection when spending from Whirlpool Postmix wallet

### ISSUE THREAD

**Issue by alkum** [sparrowwallet/sparrow#424]
In the Postmix UTXO view, selecting multiple UTXOs will update the bottom right button "Send Selected (XYZ sats)" based on the total amount selected.

The issue is when one of the selected UTXOs is in the process of re-mixing (e.g. has a progress bar shown in the Mixes column showing tooltips such as "Registered input (1 of xyz)").

If selecting a combination of UTXOs "not currently re-mixing" (without progress bar) and some "currently re-mixing" (with progress bar), then:

- the bottom right button Send Selected updates to show total of "not currently re-mixing" UTXOs
- clicking it goes to the next screen (Send) where
  - the pre-filled amount is a total of both kinds of selected UTXOs
  - the button "Max (`X` UTXOs selected)" shows `X` as if both kinds of UTXOs are considered

So to summarize, in the first screen the Send Selected button gives the impression that only UTXOs "not currently re-mixing" are considered for sending. But the second screen shows values as if all selected UTXOs are considered.

**Comment by craigraw** [comment]
Thanks for the bug report, fixed in cc31b5b. Mixing UTXOs should reflect in the Send Selected Total.
